tools/bcl-test-importer/BCLTestImporter/BCLTestAssemblyDefinition.cs

```diff
@@ -31,6 +31,18 @@ public BCLTestAssemblyDefinition (string name)
 			// assemblies
 			IsXUnit = name.Contains ("_xunit-test");
 		}
+		
+		public string GetName (Platform platform)
+		{
+			switch (platform) {
+			case Platform.WatchOS:
+				return Name.Replace ("monotouch_", "monotouch_watch_");
+			case Platform.TvOS:
+				return Name.Replace ("monotouch_", "monotouch_tv_");
+			default:
+				return Name;
+			}
+		}
```

Comment:
Needed so that we do not have to modify the project definitions. Use iOS as the default.